scene/gui/split_container.cpp

```diff
@@ -325,25 +335,23 @@ void SplitContainer::clamp_split_offset() {
 	if (!_get_sortable_child(0) || !_get_sortable_child(1)) {
 		return;
 	}
-
-	_compute_middle_sep(true);
+	_compute_split_offset(true);
 	queue_sort();
 }
 
 void SplitContainer::set_collapsed(bool p_collapsed) {
 	if (collapsed == p_collapsed) {
 		return;
 	}
-
 	collapsed = p_collapsed;
+	collapsed ? dragging_area_control->hide() : dragging_area_control->show();
```

Comment:
```suggestion
```

The queue_sort will take care of it anyway, and this doesn't take into account child count.

If it does need to be here, it could be something like:
`dragging_area_control->set_visibility(!collapsed && _get_sortable_child(1));`

Reply:
I removed the line here, and tweaked `_resort()` to hide the dragger if collapse is enabled, which it wasn't doing previously.